os/drivers/kernel_test/kernel_test_drv.c

```diff
@@ -180,6 +180,12 @@ static int kernel_test_drv_ioctl(FAR struct file *filep, int cmd, unsigned long
 	case TESTIOC_KTHREAD_STACK_PROTECTION_TEST:
 		ret = test_kthread_stack_overflow_protection(cmd, arg);
 		break;
+#endif
+	/* Run the test case for pbuf   */
```

Comment:
Could you move this comment to after line 185 please since the testcase can only be executed if the CONFIG is defined.

Reply:
done